07-sampling.Rmd

```diff
@@ -663,111 +631,87 @@ As we observed in Figure \@ref(fig:comparing-sampling-distributions), as the sam
-1. A **point estimate**, also known as a **sample statistic**, is a summary statistic computed from a sample that *estimates* the unknown population parameter.
-
-So previously we conducted **sampling** using a shovel with 50 slots to extract samples of size $n$ = 50. To perform the virtual analog of this sampling, recall that we used the `rep_sample_n()` function as follows:
-
-```{r, eval = FALSE}
-virtual_shovel <- bowl %>% 
-  rep_sample_n(size = 50)
-virtual_shovel
-```
-```{r, echo = FALSE}
-virtual_shovel
-```
-
-Using the sample of 50 balls contained in `virtual_shovel`, we generated an estimate of the proportion of the bowl's balls that are red `prop_red`
+1. **Sampling** is the act of collecting samples from the population and using them to estimate population parameters. We mathematically denote the sample size using lowercase $n$. Typically the sample size $n$ is much smaller than the population size $N$. Thus taking a sample or a few samples is easier and cheaper than completing a census.
+      - All our bowl activities, tactile and virtual, are examples of sampling.
+1. A **sample statistic** or simply a **statistic** is a numerical summary, a number, computed from a sample. When this statistic is used to estimate a *population parameter* we also call it a **point estimate**.
+    - Two point estimates that are often used are the proportion and average (or mean) of the sample. We call them *sample proportion* and *sample mean* and use $\widehat{p}$ and $\widehat {\mu}$ to denote them, respectively. The hat above the letter is a common convention in statistics to differentiate point estimates (samples) from parameters (populations).
 
+As we did in our virtual sampling activity, we take a random sample of $n$=50 balls from the virtual `bowl` and use it to obtain the sample proportion of red balls, $\widehat{p}$. We use the same `dplyr` verbs used earlier, as shown below:
 
 ```{r}
-virtual_shovel %>% 
+bowl %>%
+  rep_sample_n(size = 50) %>% 
   summarize(num_red = sum(color == "red")) %>% 
   mutate(prop_red = num_red / 50)
 ```
 
-So in our case, the value of `prop_red` is the **point estimate** of the population proportion $p$ since it estimates the latter's value. Furthermore, this point estimate has a specific name when considering proportions: the *sample proportion*. It is denoted using $\widehat{p}$ because it is a common convention in statistics to use a "hat" symbol to denote point estimates.
+The sample proportion, $\widehat{p}$, is given by the value of `prop_red`. It is also called a point estimate because is used to estimate the population proportion, $p$.
```

Comment:
```suggestion
The sample proportion, $\widehat{p}$, is given by the value of `prop_red`. It is also called a point estimate because it is used to estimate the population proportion, $p$.
```